ISISComputingGroup/EPICS-ioc
# Added files for SKF IOC 06

### Description of work

Additional SKFCHOPPER IOC required for LET as chopper 1 has recently been upgraded to SKF control.

### To test

Build IOC and check 06 exists.

Unable to test with hardware at present due to ongoing commissioning.

### Acceptance criteria

Successful build of sixth IOC and associated support structure.

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
